src/System.Net.Sockets/src/System/Net/Sockets/SocketPal.Unix.cs

```diff
@@ -890,8 +892,8 @@ public static SocketError Send(SafeCloseSocket handle, IList<ArraySegment<byte>>
             int bufferIndex = 0;
             int offset = 0;
             SocketError errorCode;
-            bool completed = TryCompleteSendTo(handle, bufferList, ref bufferIndex, ref offset, socketFlags, null, 0, ref bytesTransferred, out errorCode);
-            return completed ? errorCode : SocketError.WouldBlock;
+            TryCompleteSendTo(handle, bufferList, ref bufferIndex, ref offset, socketFlags, null, 0, ref bytesTransferred, out errorCode);
+            return errorCode;
```

Comment:
Is the return value of TryCompleteSendTo used anywhere anymore?

Reply:
The async code uses it to know when the operation has completed.